modules/references-regions-zones-infrastructure-vsphere.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+// * post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc
+
+:_content-type: REFERENCE
+[id="references-regions-zones-infrastructure-vsphere_{context}"]
+= Parameters for the `infrastructures.config.openshift.io` CRD
+
+You must set values for specific parameters in the `infrastructures.config.openshift.io` Custom Resource Definition (CRD) to define multiple regions and zones for your {product-title} cluster that runs on a VMware vSphere instance. For an example of a `infrastructures.config.openshift.io` configuration, see xref:../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#specifying-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration[Specifying multiple regions and zones for your cluster on vSphere].
+
+The following table lists mandatory parameters for defining multiple regions and zones for your {product-title} cluster:
+
+[cols="1,2",options="header"]
+|===
+|Parameter | Description
+
+|`vcenters` | Lists the vCenter server for your {product-title} cluster. You can only specify one vCenter for your cluster.
+|`datacenters` | Lists vCenter datacenters where VMs associated with the {product-title} cluster will be created or presently exist.
+|`port` | The TCP port of the vCenter server.
+|`server` | The FQDN of the vCenter server.
```

Comment:
I'd prefer to see FQDN spelled out. I suppose an argument can be made that it's common enough that everyone knows it, but I remember being in a meeting where (not me for a change) asked what FQDN meant. I figured if that guy didn't know, it was worth me spelling out. Just something to think about.